README.md

```diff
@@ -191,6 +191,11 @@ Edit `AppDelegate.m`:
 +   [[UNUserNotificationCenter currentNotificationCenter] setDelegate:self];
 + }
 +
+// Required to register for notifications
++ - (void)application:(__unused UIApplication *)application didRegisterUserNotificationSettings:(UIUserNotificationSettings *)notificationSettings
```

Comment:
will firebase's swizzle keep working after you defining this method?
or say, for a new device, will ios notification token still get associated with firebase id automatically?